Update roadmap.rst (#107)
FirelyTeam/firely-docs-firely-server: releasenotes/roadmap.rst

Disclaimer: No rights can be derived from this roadmap.

2022
----

Q2
^^

> * Various performance improvments and adoption of features currently only available for SQL Server on MongoDB

Reviewer: ```suggestion
* Addition of various performance improvements and features to our MongoDB implementation which are currently only available for SQL Server
```

  
  * Support for transactions 

Q3
^^

* Check compliance with CMS ImplementationGuides

* Bulk data export as a separate service - deploy and scale it independently of Firely Server itself

> * New Firely Validator as a separate service - deploy and scale it independently of Firely Server itself

Reviewer: ```suggestion
* New Firely Validator as a separate service - deploy and scale it independently of Firely Server
```

> * Provide production-ready IdentityServer instance bundled together with Firely Server

Reviewer: ```suggestion
* Provide production-ready IdentityServer instance bundled with Firely Server
```
